.github/workflows/ci.yml

```diff
@@ -43,6 +52,8 @@ jobs:
       - uses: julia-actions/julia-buildpkg@v1
       - uses: julia-actions/julia-runtest@v1
       - uses: julia-actions/julia-processcoverage@v1
+        continue-on-error: true
```

Comment:
What's the advantage of this? I'd rather be warned when coverage reporting failed.

Reply:
As mentioned in https://github.com/JuliaMath/SpecialFunctions.jl/pull/357#discussion_r735590713, it can be helpful to let the coverage upload etc. not influence the Github test status. Lately, I also had some problems with coverage uploads since the coverage services were down and it is annoying if this causes test failures. One could still make the coverage checks mandatory on Github to prevent merging PRs with significant coverage drops.

I can revert the change if not desired though, I just used the same settings as in SpecialFunctions.

Reply:
I find it useful to get a big red icon when coverage failed for some reason. That gives you a hint that something might be wrong. You can still ignore it if it turns out to be wrong, but sometimes it catches real lacks in coverage.

Reply:
I don't mind, so I'll just revert these changes.

Reply:
Done.